### PR TITLE
[dev-v5] Add FluentText component

### PR DESCRIPTION
**Pull request by vnbaaij**
Adds the FluentText component. This is a replacement for the v4 Label component.

image

Unit Test added with 100% line coverage

### CODE REVIEW

File: src/Core/Microsoft.FluentUI.AspNetCore.Components.csproj [hunk 1]
```diff
@@ -115,5 +115,8 @@
       <LastGenOutput>%(ParentFile).Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="Components\Text\" />
```
Comment (dvoituron) [inline src/Core/Microsoft.FluentUI.AspNetCore.Components.csproj:119]: Can you remove this line?

File: src/Core/Components/Text/FluentText.razor.cs [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    /// Gets or sets the font family of the text.
+    /// </summary>
+    [Parameter]
+    public TextFont? Font { get; set; }
+
+    /// <summary>
+    /// Gets or sets if the texts is set to no-wrap.
+    /// </summary>
+    [Parameter]
+    public bool Nowrap { get; set; }
```
Comment (dvoituron) [inline src/Core/Components/Text/FluentText.razor.cs:58]: Can you avoid the "negation" of properties?
And to use "Wrap" like did by [React](https://react.fluentui.dev/?path=/docs/components-text--docs)
(and with default value to `True`)

File: tests/Core/Microsoft.FluentUI.AspNetCore.Components.Tests.csproj [hunk 3]
```diff
@@ -58,4 +58,8 @@
       <DependentUpon>%(ParentFile).razor</DependentUpon>
     </None>
   </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Components\Text\" />
```
Comment (dvoituron) [inline tests/Core/Microsoft.FluentUI.AspNetCore.Components.Tests.csproj:63]: Can you remove this line?

**Review by dvoituron — CHANGES_REQUESTED** [review]

File: src/Core/Components/Text/FluentText.razor.cs (hunk 2 above)
Comment (vnbaaij) [reply src/Core/Components/Text/FluentText.razor.cs:58]: This is the name of the attribute in the web componnets. Not a fan either but think it is better to keep it the same

**Review by dvoituron — APPROVED** [review]

**Pull request merged**